docs/Install.md

```diff
@@ -1,22 +1,125 @@
 
 # Installation
 
-The following will walk through the deployment of Pelorus.
+There are three methods to deploy Pelorus:
+
+(Recommended) First two are using Pelorus Operator that is available by default in the [Red Hat-provided community-operators catalog sources](https://docs.openshift.com/container-platform/4.11/operators/understanding/olm-rh-catalogs.html#olm-rh-catalogs_olm-rh-catalogs). This catalog is distributed as part of the OpenShift:
```

Comment:
Instead of adding "Recommended" here, I would add "Deprecated" to helm

Something like

There are three methods to deploy Pelorus:

- Using Pelorus Operator that is available by default in the [Red Hat-provided community-operators catalog sources](https://docs.openshift.com/container-platform/4.11/operators/understanding/olm-rh-catalogs.html#olm-rh-catalogs_olm-rh-catalogs)

    - OpenShift [Command Line Tool](https://pelorus--767.org.readthedocs.build/en/767/Install/#openshift-command-line-tool)
    - OpenShift [Dashboard](https://pelorus--767.org.readthedocs.build/en/767/Install/#openshift-dashboard)
- (Depracated) [Helm charts](https://pelorus--767.org.readthedocs.build/en/767/Install/#helm-charts)